Library/ViewModels/DiscoveryPostcardViewModel.swift

```diff
@@ -273,6 +395,33 @@ public final class DiscoveryPostcardViewModel: DiscoveryPostcardViewModelType,
   public var outputs: DiscoveryPostcardViewModelOutputs { return self }
 }
 
+private func cached(project: Project) -> Project {
+  if let projectCache = AppEnvironment.current.cache[KSCache.ksr_projectSaved] as? [Int: Bool] {
+    let isSaved = projectCache[project.id] ?? project.personalization.isStarred
+    return project |> Project.lens.personalization.isStarred .~ isSaved
+  } else {
+    return project
+  }
+}
+
+private func cache(project: Project, shouldToggle: Bool) -> Bool {
```

Comment:
I think there's some value in adding a little comment explaining what's happening in this function